tensorrt_llm/_torch/modules/kimi_k3_moe/_moe_kernels.py

```
@@ -0,0 +1,412 @@
+    return (x + alignment - 1) // alignment * alignment
+
+
+def get_moe_sm_version() -> int:
+    """Return the runtime SM version used for kernel-support checks."""
+    if not torch.cuda.is_available() or torch.cuda.device_count() == 0:
+        return -1
+    try:
+        from tensorrt_llm._utils import get_sm_version as _tllm_get_sm_version
+
+        return int(_tllm_get_sm_version())
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)